pkgs/development/tools/gomarkdoc/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, buildGoModule, fetchFromGitHub, testers, gomarkdoc }:
+
+buildGoModule rec {
+  pname = "gomarkdoc";
+  version = "1.1.0";
+  src = fetchFromGitHub {
+    owner = "princjef";
+    repo = "gomarkdoc";
+    rev = "v${version}";
+    sha256 = "sha256-eMH+F1ZXAKHqnrvOJvCETm2NiDwY03IFHrDNYr3jaW8=";
```

```suggestion
    hash = "sha256-eMH+F1ZXAKHqnrvOJvCETm2NiDwY03IFHrDNYr3jaW8=";
```
the sha256 key is more or less deprecated and you should use hash instead.